packages/reserve-site/src/components/Chevron.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+import colors from './colors'
+
+export enum Direction {
+  right = 0,
+  down = 90,
+  left = 180,
+  up = 270,
+}
+
+interface Props {
+  size?: number | string
+  direction?: Direction
+}
+
+export default class Chevron extends React.PureComponent<Props> {
+  static defaultProps = {
```

Comment:
you can still make this a function if you want: https://stackoverflow.com/questions/41488537/react-defaultprops-vs-es6-default-params-when-destructuring-performances-issu